packages/gasket-plugin-intl/lib/build-modules.js

```diff
@@ -126,11 +133,11 @@ class BuildModules {
   /**
    * Processes directories
    *
-   * @param {string[]} buildDirs - list of dirs to process
+   * @param {SrcPkgDir[]} srcPkgDirs - list of dirs to process
    */
-  async processDirs(buildDirs) {
-    for (const srcDir of buildDirs) {
-      const pkgName = await this.getPackageName(srcDir);
+  async processDirs(srcPkgDirs) {
+    for (const srcPkgDir of srcPkgDirs) {
+      const [pkgName, srcDir] = srcPkgDir;
```

Comment:
```suggestion
    for (const [pkgName, srcDir] of srcPkgDirs) {
```
Bad idea to consolidate onto one line?